### PR TITLE
Fixed evaporator liquid rendering

### LINKED ISSUE #4425: Thermal Evaporation Plant causes massive lag while entering world
It is caused by instancing many display lists during recalculation of water model inside the plant.

Reproduction: Add one level to fully built and filled plant

### PR DESCRIPTION
**Pull request by PlayerWithoutName**
Simplified it a little and fixed it's lagginess, now it generates max 17 Display Instances per fluid ;)

Fixes #4425

### CODE REVIEW

**Comment by thommy101** [comment]
If this fixes what I think it fixes, it would be great! Adding layers causes so much lag right now (full stops for 3 seconds)

**Comment by PlayerWithoutName** [comment]
Yeah, you will probably adapt a little but main idea was not to generate so much instances as on my processor it was taking about 10 seconds to unfreeze after joining and i had to disassemble Evaporation Plants because my friend's laptop was timing out before these instances could generate.

File: src/main/java/mekanism/client/render/tileentity/RenderThermalEvaporationController.java [hunk 1]
```diff
@@ -74,118 +105,67 @@ private void push()
 		GL11.glBlendFunc(GL11.GL_SRC_ALPHA, GL11.GL_ONE_MINUS_SRC_ALPHA);
 	}
 
+
+
 	@SuppressWarnings("incomplete-switch")
```
Comment (thommy101) [inline src/main/java/mekanism/client/render/tileentity/RenderThermalEvaporationController.java]: Since the switch has been removed, this is obsolete

**Comment by Meller127** [comment]
Will this be pushed to 1.10 branch also?

**Comment by PlayerWithoutName** [comment]
Fixed fluid caching as i've realized that i forgot to cache lists after generating

**Comment by thiakil** [comment]
Can someone explain to me how the rendering is supposed to look? (or currently looks at least)

I know that looking down from above you see the top of the fluid at the relevant level, but how does it look when standing inside the tower?

**Comment by thommy101** [comment]
Top, bottom looking up, half way looking down

<details>
  <summary>Click to expand</summary>
Top
Bottom looking up
Half way looking down
</details>

**Comment by aidancbrady** [comment]
Thanks!

**Pull request merged**